### PR TITLE
fix trt multiclass_nms3

### PR DESCRIPTION
**Pull request by zhangjun**
### PR types

Others
### PR changes

Others
### Describe

- fix multiclass_nms3 for static shape
yolov3_mobilenet_v3_large_270e_coco, bs=1, T4

   |before|after|
   |-|-|
   |43.0865|15.193|

- dynamic shape support
   - dynamic
      bbox: [batch_size, num_boxes, num_classes, 4] or [batch_size, num_boxes, 1, 4]
      score: [batch_size, num_boxes, num_classes] or [batch_size, num_boxes, num_classes, 1]

   |before|after|
   |-|-|
   |48.906|15.3634|

### CODE REVIEW

**Review by wangxinxin08 — COMMENTED** [review]
LGTM

File: paddle/fluid/inference/tensorrt/op_teller.cc [hunk 1]
```diff
@@ -340,9 +342,9 @@ bool OpTeller::Tell(const framework::ir::Node* node,
     if (!with_dynamic_shape) {
       std::string X_name;
       auto inputs = desc.Inputs();
-      if (inputs.count("X") && !desc.Input("X").empty()) {
```
Comment (cyj1986) [inline paddle/fluid/inference/tensorrt/op_teller.cc:343]: 去掉判空条件的原因是什么

**Review by jerrywgz — APPROVED** [review]

**Review by cyj1986 — COMMENTED** [review]
判空的部分请再确认一下

File: paddle/fluid/inference/tensorrt/op_teller.cc (hunk 1 above)
Comment (zhangjun) [reply paddle/fluid/inference/tensorrt/op_teller.cc:343]: 已修改

**Pull request merged**